src/histolab/util.py

```diff
@@ -117,6 +117,25 @@ def polygon_to_mask_array(dims: tuple, vertices: CoordinatePair) -> np.ndarray:
     return np.array(img).astype(bool)
 
 
+def random_choice_true_mask2d(binary_mask: np.ndarray) -> Tuple[int, int]:
+    """Return a random pair of indices where the ``binary_mask`` is True.
+
+    Parameters
+    ----------
+    binary_mask : np.ndarray
+        Binary array.
+
+    Returns
+    -------
+    Tuple[int, int]
+        Random pair of indices where the ``binary_mask`` is True.
+    """
+    x = np.random.choice(np.where(binary_mask)[0])
+    y = np.random.choice(np.where(binary_mask[x])[0])
+
+    return x, y
```

Comment:
I don't see any reason of a new util functions that is used basically in one single place. If in future we need this in more than 1 code portion so we can introduce it in the `util` module.

Reply:
alright, should it be a private static method of `RandomTiler` then?

Reply:
for 2 lines and then used in only one part in the randomtiler? i don't think so just copy and paste that 2 lines when you need those :)

Reply:
mmm not so easily testable in that case

Reply:
why? you could check the call args and assert that the y has the correct value you are expecting and that depends on the x.

```
scale_coordinates(
            reference_coords=CoordinatePair(x_ul_lvl, y_ul_lvl, x_br_lvl, y_br_lvl),
            reference_size=binary_mask.shape[::-1],
            target_size=slide.dimensions,
)
```

during the test you can assert that the scale_coordinates has been called with the right x_ul_lvl and y_ul_lvl .. do you?

Reply:
in the test I want to check that the coordinates I sampled correspond to a True value in the mask, and I won't be checking that with your proposal...